plugin/src/main/kotlin/org/gradleweaver/plugins/jlink/Endianness.kt

```diff
@@ -0,0 +1,21 @@
+package org.gradleweaver.plugins.jlink
+
+/**
+ * Options for the available byte orders of generated runtime images.
+ */
+enum class Endianness {
+    /**
+     * Use the endianness of the build system.
+     */
+    SYSTEM_DEFAULT,
```

Comment:
Shouldn't we just use the system default by default? Or is this because we can't determine the system default?

Reply:
This is equivalent to not passing the `--endian` flag. The `jlink` tool will use the byte order of the build system.

Reply:
Okay.